ansible/library/conn_graph_facts.py

```diff
@@ -349,6 +349,7 @@ def csv_to_graph_facts(self):
                     "peerport": entry["StartPort"],
                     "proxy": entry["Proxy"],
                     "type": entry["Console_type"],
+                    "menu_type": entry["Console_menu_type"],
```

Comment:
@matthew-soulsby please also give an update for the example file `ansible/files/sonic_lab_console_links.csv`

Reply:
Good point, added the column with empty values (to leave behaviour as it was prior to the changes), how does that look?